lib/wpc_proto/proto_options/wp_global.options

```diff
@@ -1,5 +1,5 @@
 wp.RequestHeader.sink_id max_size:16
-wp.ResponseHeader.gw_id max_size:16
+wp.ResponseHeader.gw_id max_size:32
 wp.ResponseHeader.sink_id max_size:16
-wp.EventHeader.gw_id max_size:16
+wp.EventHeader.gw_id max_size:32
 wp.EventHeader.sink_id max_size:16
```

Comment:
are we sure we don't want to increase sink_id size to 32 bytes at the same time ? (possible, I just prefer to ask )

Reply:
SinkId should be kept as small as possible. We could/should maybe just hardcode it to "0" for example as we target only one sink gateway.